CogniteSdk/test/fsharp/TimeSeries.fs

```diff
@@ -386,3 +386,23 @@ let ``Update timeseries is Ok`` () = task {
 
     // Assert delete
 }
+
+[<Fact>]
+let ``Synthetic Query is Ok`` () = task {
+    // Arrange
+    let query =
+        TimeSeriesSyntheticQuery(
+            Items = [
+                TimeSeriesSyntheticQueryItem(Expression="ts{externalId='pi:160627'} + 1", Limit = Nullable 10)
+            ]
+        )
+
+    // Act
+    let! res = readClient.TimeSeries.SyntheticQueryAsync(query)
+
+    let ts = res |> Seq.head |> (fun x -> x.DataPoints)
+
+    // Assert
```

Comment:
Might want to check that ts is not empty
test <@ ts |> Seq.length > 0 @>

Reply:
I cannot be sure about that, since it's publicdata.
Maybe I should create a ts a push some dps as a part of the test?